pkg/scheduler/scheduler.go

```diff
@@ -311,9 +306,6 @@ func initPolicyFromConfigMap(client clientset.Interface, policyRef *schedulerapi
 
 // Run begins watching and scheduling. It waits for cache to be synced, then starts scheduling and blocked until the context is done.
 func (sched *Scheduler) Run(ctx context.Context) {
-	if !cache.WaitForCacheSync(ctx.Done(), sched.scheduledPodsHasSynced) {
```

Comment:
we block for shared informers to sync before this point, right?

Reply:
yes, that's in cmd/kube-scheduler/app/server.go#Run